tasks/move-granules/schemas/config.json

```diff
@@ -85,6 +85,18 @@
     "s3MultipartChunksizeMb": {
       "type": ["number", "null"],
       "description": "S3 multipart upload chunk size in MB.  If none is specified, the default default_s3_multipart_chunksize_mb is used."
+    },
+    "checkCrossCollectionCollisions": {
```

Comment:
If it's a requirement to check, why do we want it to be configurable?
Also need to add default value here.